packages/utilities-deprecated/projects/utilities/CHANGELOG.md

```diff
@@ -9,6 +9,706 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+  }
+}
+```
+
+- `p-title-large` mixin is now renamed to `pds-display-large` and in addition we provide `pds-display-medium`. The size
+  of the new `display` styles are changing fluid depending on the viewport width instead of fixed predefined sizes on
+  specific breakpoints, and they are italic now.
+
+```diff
+- @import '~@porsche-design-system/utilities/scss';
++ @import '~@porsche-design-system/components-{js|angular|react|vue}/styles/scss';
+
+h1 {
+- @include p-title-large;
++ @include pds-display-large;
+}
+
+h2 {
++ @include pds-display-medium;
+}
+```
+
+- `p-headline-{1|2|3|4|5}` got renamed/extended to `pds-heading-{small|medium|large|x-large|xx-large|xxx-large}`. The
+  size of the new `heading` styles are changing fluid depending on the viewport width instead of fixed predefined sizes
+  on specific breakpoints, and they are italic now.
+
+```diff
+- @import '~@porsche-design-system/utilities/scss';
++ @import '~@porsche-design-system/components-{js|angular|react|vue}/styles/scss';
+
+h1 {
+- @include p-headline-1;
++ @include pds-heading-xx-large;
+}
+
+h2 {
+- @include p-headline-2;
++ @include pds-heading-x-large;
+}
+```
+
+- `p-text-{x-small|small|medium|large|x-large}` got renamed to `pds-text-{x-small|small|medium|large|x-large}`. Except
+  `pds-text-small` all text styles have fluid sizes that change depending on the viewport width instead of fixed
+  predefined sizes on specific breakpoints.
+
+```diff
+- @import '~@porsche-design-system/utilities/scss';
++ @import '~@porsche-design-system/components-{js|angular|react|vue}/styles/scss';
+
+p {
+- @include p-text-small;
++ @include pds-text-small;
+}
+```
+
+**Colors**
+
+As of v3 of the Porsche Design System the color theme got reworked completely to achieve a monochrome look. To get an
+overview ot the new colors have a look [here](https://designsystem.porsche.com/latest/styles/theme).
+
+- `brand` got renamed to `primary`.
+
+```diff
+- @import '~@porsche-design-system/utilities/scss';
++ @import '~@porsche-design-system/components-{js|angular|react|vue}/styles/scss';
+
+div {
+- background: {$p-color-brand|$p-color-theme-light-brand};
++ background: $pds-theme-light-primary;
+}
+```
+
+- `default` is removed, use `primary` instead.
+
+```diff
+- @import '~@porsche-design-system/utilities/scss';
++ @import '~@porsche-design-system/components-{js|angular|react|vue}/styles/scss';
+
+div {
+- color: {$p-color-default|$p-color-theme-light-default};
++ color: $pds-theme-light-primary;
+}
+```
+
+- `$p-color-theme-light-background` is renamed to `$pds-theme-light-background-base`.
+
+```diff
+- @import '~@porsche-design-system/utilities/scss';
++ @import '~@porsche-design-system/components-{js|angular|react|vue}/styles/scss';
+
+div {
+- background: {$p-color-background-default|$p-color-theme-light-background};
++ background: $pds-theme-light-background-base;
+}
+```
+
+- to keep it clear, only the old color variable in regard to its new name will be mentioned below:
```

Comment:
Maybe better:
For the sake of clarity, only the old color variables are mentioned below in relation to their new names.